### DISCUSSION TITLE
Music attempt

### DISCUSSION THREAD

**Discussion by V-ktor** [redeclipse/base#1130] [Maps and Content Showcase]
Recently I started creating music. I don't have much knowledge about music in general, just a bit music theory.
I tried to copy the style of the existing music and I called the files optimistically 'track_13' and 'track_14'.
Well, I don't really like it, but it's probably because they didn't turned out like intended. It's not what I had in mind, not even similar at all. I don't know how that happened.
I need help here.
Here are ogg files and lmms source files:
[music.zip](https://github.com/red-eclipse/world/files/1689986/music.zip)
[lmms.zip](https://github.com/red-eclipse/world/files/1690078/lmms.zip)
License is CC-BY-SA 3.0.

**Comment by nok-ko** [comment]
Both sound like the starts of some very usable tracks. (to nokko's ears, anyway.) With more work, I could see something like them in the game.
